pages/sign-in/kakao/index.page.tsx

```diff
@@ -0,0 +1,18 @@
+import { GetServerSidePropsContext } from "next";
+
+import { instance } from "@/apis/axios";
+
+export async function getServerSideProps(context: GetServerSidePropsContext) {
+  const code = context.query["code"];
+
+  const res = await instance.get(`/login/oauth/kakao?code=${code}`);
```

Comment:
```suggestion
  const res = await instance.get(`/login/oauth/kakao?code=${code}`);
```
어제 디스코드 봤을 때 동일한 인가 코드로 요청하면 에러가 생긴다고 했던 것 같은데 여기 페이지로 와서 새로고침하면 에러가 생기나요??

Reply:
네 맞습니다! 해당 페이지에서 새로고침 시 에러가 발생합니다...😢

Reply:
음 그러면 `tanstack query`로 5분정도 캐싱하고 그 이후에 새로고침해서 에러가 발생하면 다시 로그인 페이지로 이동시키는 방식으로 하던지 해야할 것 같네요!

Reply:
해당 url로의 이동과 동시에 api 요청이 들어가면서 에러가 나는거라, 캐싱없이 에러 발생하면 그냥 로그인 페이지로 이동시키는 것도 괜찮을 것 같긴 하네용

Reply:
근데 생각해보니 문제가 있는게, 중복 인가 코드로 요청했을 때 돌아오는 상태 코드가 500이라,

상태코드가 500일 때 로그인 페이지로 보내는 로직을 만들어버리면,

진짜 서버 상에서 문제가 터졌을 때 무한 도르마무를 하게 될 수도 있을 것 같다는 생각이 드네요...? 😭😭😭

Reply:
백엔드분들한테 상태코드를 바꿔달라고 하면 될 것 같아요!